### PR TITLE
Modal feature

### PR DESCRIPTION
**Pull request by Jason247Miller**
Note: I am trying to find the right ESlint rules to be applied to this entire project, so that I can simply do --fix and then 
be done. 

There is a lot of indentation changes on this commit because of this.

### CODE REVIEW

**Comment by Jason247Miller** [comment]
the update() method in the daily-review.ts file I had some difficulty with. I really wanted to use a forkjoin to ensure that
the modal does not close until the subscribe is complete, but I could not get it to work as desired.

File: src/app/components/daily-review/daily-review.component.html [hunk 1]
```diff
@@ -1,30 +1,79 @@
...
+                        <td>
+                          <a class="delete" title="Delete" (click)="deleteItem(i)" data-toggle="tooltip"><i
+                              class="material-icons">&#xE872;</i></a>
+                        </td>
+                      </tr>
+                    </ng-container>
+                  </ng-container>
+                  <tr>
+                    <td>
+                      <input type="text" formControlName="addItemInput" style="width:100%">
```
Comment (swhook52) [inline src/app/components/daily-review/daily-review.component.html:50]: Avoid inline styles. Keep them in CSS files.

File: src/app/components/daily-review/daily-review.component.html [hunk 2]
```diff
@@ -1,30 +1,79 @@
...
+              <div class="table-title">
+                <div class="row">
+                  <div class="col-sm-8">
+                    <h2></h2>
+                  </div>
+                  <div class="col-sm-4">
+                  </div>
+                </div>
+              </div>
+              <table class="table table-bordered">
```
Comment (swhook52) [inline src/app/components/daily-review/daily-review.component.html:22]: Modern pages don't use tables for layout. I'm hoping the CSS course work I shared with you will elaborate on this more.

**Comment by swhook52** [comment]
Still a lot of indentation and spacing problems. may want to start a habit of using `shift + alt + f` when you enter/exit a file. It will format based on your rules. Consistency in formatting is important for maintainability and readability.

**Review by swhook52 — APPROVED** [review]

**Comment by swhook52** [comment]
I approved this because the comments are minor but there are merge conflicts. Don't worry about this PR until after the CSS course.

**Pull request closed without merging**